typescript/web/src/utils/setup-local-db-tests.ts

```diff
@@ -4,6 +4,7 @@ import { initMockedDate } from "@labelflow/dev-utils/mockdate";
 import { getDatabase } from "../connectors/database";
 import { client } from "../connectors/apollo-client/schema-client";
 
+// TODO SW probably obsolete and to delete
 export function setupTestsWithLocalDatabase() {
```

Comment:
- [ ] We have to fix this TODO in this PR. But I agree it can be delayed to the end

Reply:
Yes my intention is do cleanup this code and other similar ones in this PR, but only after all tests are running.